mmv1/products/cloudbuild/api.yaml

```diff
@@ -28,6 +28,138 @@ apis_required:
+        url_param_only: true
+        input: true
+        required: true
+        description: |
+          The location of this bitbucket server config.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        output: true
+        description: |
+          The resource name for the config.
+      - !ruby/object:Api::Type::String
+        name: 'hostUri'
+        required: true
+        description: |
+          Immutable. The URI of the Bitbucket Server host. Once this field has been set, it cannot be changed.
+          If you need to change it, please create another BitbucketServerConfig.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'secrets'
+        required: true
+        description: |
+          Secret Manager secrets needed by the config.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'adminAccessTokenVersionName'
+            required: true
+            description: |
+              The resource name for the admin access token's secret version.
+          - !ruby/object:Api::Type::String
+            name: 'readAccessTokenVersionName'
+            required: true
+            description: |
+              The resource name for the read access token's secret version.
+          - !ruby/object:Api::Type::String
+            name: 'webhookSecretVersionName'
+            required: true
+            input: true
+            description: |
+              Immutable. The resource name for the webhook secret's secret version. Once this field has been set, it cannot be changed.
+              If you need to change it, please create another BitbucketServerConfig.
+      - !ruby/object:Api::Type::Time
+        name: 'createTime'
+        output: true
+        description: |
+          Time when the config was created.
+      - !ruby/object:Api::Type::String
+        name: 'username'
+        required: true
+        description: |
+          Username of the account Cloud Build will use on Bitbucket Server.
+      - !ruby/object:Api::Type::String
+        name: 'webhookKey'
+        output: true
+        description: |
+          Output only. UUID included in webhook requests. The UUID is used to look up the corresponding config.
+      - !ruby/object:Api::Type::String
+        name: 'apiKey'
+        required: true
+        input: true
+        description: |
+          Immutable. API Key that will be attached to webhook. Once this field has been set, it cannot be changed.
+          If you need to change it, please create another BitbucketServerConfig.
+      - !ruby/object:Api::Type::Array
+        name: 'connectedRepositories'
+        description: |
+          Connected Bitbucket Server repositories for this config.
+        item_type: !ruby/object:Api::Type::NestedObject
+          name: 'BitbucketServerRepositoryId'
+          description: |
+            BitbucketServerRepositoryId identifies a specific repository hosted on a Bitbucket Server.
```

Comment:
I think we can remove these two lines. Don't think they will get generated.

Reply:
Ah, yes, you are right. I assume you mean name as well? It looks like name and description are not generated. Removing both.